stdlib/dict/dict.flux

```diff
@@ -1,20 +1,130 @@
 package dict
```

Comment:
Can you add a package doc comment? https://docs.influxdata.com/influxdb/cloud/reference/flux/stdlib/dict/